### PR TITLE
Fix integration test

### PR DESCRIPTION
**Pull request by bhautikpip**
1. Fixes integration test failures
2. cleaned up sample app and README

### CODE REVIEW

File: .github/collector/docker-compose.yml [hunk 1]
```diff
@@ -10,4 +10,4 @@ services:
       AWS_SECRET_ACCESS_KEY: ${SECRET_ACCESS_KEY}
     ports:
       - '55680:55680'
-      - '30080:30080'
+      - '9464:9464'
```
Comment (anuraaga) [inline .github/collector/docker-compose.yml:13]: What's this port?

File: .github/collector/otel/collector-config.yml [hunk 2]
```diff
@@ -2,37 +2,30 @@ receivers:
   otlp:
     protocols:
       grpc:
+        endpoint: 0.0.0.0:55680
```
Comment (anuraaga) [inline .github/collector/otel/collector-config.yml:5]: The new port for OTLP is 4317, it'd be good to update, but can be another PR too

File: .github/workflows/integration-test.yml [hunk 3]
```diff
@@ -50,19 +50,16 @@ jobs:
           ACCESS_KEY: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
```
Comment (anuraaga) [inline .github/workflows/integration-test.yml:50]: Can we instead use the AWS_ACCESS_KEY_ID here so there's only one account?

File: .github/workflows/integration-test.yml [hunk 4]
```diff
@@ -74,7 +71,7 @@ jobs:
           path: test-framework
       
       - name: Run testing suite
-        run: ./gradlew :validator:run --args='-c go-otel-trace-validation.yml --endpoint http://127.0.0.1:8080 --metric-namespace aws-otel-integ-test -t ${{ github.run_id }}-${{ github.run_number }}'
+        run: ./gradlew :validator:run --args='-c go-otel-trace-validation.yml --endpoint http://127.0.0.1:8080'
```
Comment (anuraaga) [inline .github/workflows/integration-test.yml:74]: Think you need to add the `-t` or it will always pass

File: .github/collector/otel/collector-config.yml (hunk 2 above)
Comment (bhautikpip) [reply .github/collector/otel/collector-config.yml:5]: I see. I will update this in next PR since also have to update and test benchmark.

**Review by anuraaga — APPROVED** [review]
